docs/content/users/quickstart.md

```diff
@@ -1,274 +1,230 @@
 # CMS Quickstarts
 
-While the generic `php` project type is [ready to go](./project.md) with any CMS or framework, DDEV offers project types for more easily working with popular platforms and content management systems:
+With the generic `php` and `python` project types DDEV is [ready to go](./project.md) with any PHP and Python based content management system (CMS) or framework. In addition there are preconfigured project types for many popular platforms and CMSes. To learn more about how to manage projects in DDEV visit the [Managing Projects](../users/usage/managing-projects.md) page.
```

Comment:
There’s an echo in “With the generic...” and “...with any PHP...” and all of this could be more concise:

```suggestion
DDEV is [ready to go](./project.md) with generic project types for PHP and Python frameworks, and more specific project types for [working with popular platforms and CMSes](../users/usage/managing-projects.md).
```

Reply:
i like the changes for the first sentence. but i wouldnt strike the second and just leave in the link to the managing projects page. for me the managing projects page has a different purpose than what the first sentence is about. therefore i would consider it confusing. the first sentence is about what is available on the quickstarts page while the striked second sentence is about what the manageing projects page is about.

Reply:
I would suggest to bring the second sentence back in: 

```DDEV is [ready to go](./project.md) with generic project types for PHP and Python frameworks, and more specific project types for working with popular platforms and CMSes. To learn more about how to manage projects in DDEV visit the [Managing Projects](../users/usage/managing-projects.md) page.```

Reply:
It's your PR, so write it the way you think it should be and Matt will get another chance :)

Reply:
> I would suggest to bring the second sentence back in:
> (...)

This reads better to me, and I see your point! Only nitpick left is that it could end with “visit Managing Projects” since that’d clearly link to a page—meaning “the” and “page” don’t add anything and can be removed.

Reply:
pushed a commit with your suggested changes.